File: algorithms/breaking-records/breaking_records.py
def breakingRecords(score):
  last_min_score = last_max_score = score[0]
  min_score = int()
  max_score = int()

  for point in score:
    if point > last_max_score:
      last_max_score = point
      max_score += 1

    if point < last_min_score:
      last_min_score = point
      min_score += 1

  return [max_score, min_score]

File: algorithms/breaking-records/breakingRecords.php
<?php
function breakingRecords($score) {
  $lastMinScore = $lastMaxScore = $score[0];
  $minScore = $maxScore = 0;

  foreach ($score as $point) {
    if ($point > $lastMaxScore) {
      $lastMaxScore = $point;
      $maxScore++;
    }

    if ($point < $lastMinScore) {
      $lastMinScore = $point;
      $minScore++;
    }
  };

  $result = array($maxScore, $minScore);
  return $result;
}
?>
